packages/ariakit/src/tooltip/__examples__/tooltip/test.tsx

```diff
@@ -0,0 +1,19 @@
+import { getByRole, hover, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render tooltip", () => {
+  render(<Example />);
+  expect(getByRole("button")).toMatchInlineSnapshot(
+  `<button
+  aria-labelledby="r:0"
+  tabindex="0"
+  >
+  Hover on me!
+  </button>`);
+});
```

Comment:
I'm not able to figure out, why this text is failing, there isn't proper info on the console, can anyone help @diegohaz @AlexMunoz 

![image](https://user-images.githubusercontent.com/43727167/145865715-fde261f9-b10f-4670-9345-807c5595d505.png)

Reply:
@saideepesh000  It's an indentation problem. Your snapshot code should be: 
```
test("render tooltip", () => {
  render(<Example />);
  expect(getByRole("button")).toMatchInlineSnapshot(`
    <button
      aria-labelledby="r:0"
      tabindex="0"
    >
      Hover on me!
    </button>
  `);
});
```

Reply:
Oh, my bad, thanks @AlexMunoz ,I tried prettier `format on save`  but didn't work, how do u do the formatting, without manually ?

Reply:
I'm using this config in vscode 

```
  "editor.formatOnSave": true,
  "[typescriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  }
  ```
  and if you want to update the snapshots, you can use `npx jest -u`